### DISCUSSION TITLE
Ravencoin

### DISCUSSION THREAD

**Discussion by pfreak2000** [frstrtr/c2pool#13] [Ideas]
Can you create a p2pool or c2pool for Ravencoin?

**Comment by frstrtr** [comment]
We are not developing p2pool, c2pool will support most major coins, but I can't tell exact dates... Keep checking this repo...

**Comment by pfreak2000** [comment]
is the Algo Kawpow suitable for p2p?

which coins does c2pool support?
I only found DGB in the network directory

**Comment by frstrtr** [comment]
Test development goes around DGB since it is still possible to mine solo. Potentially c2pool will support any Proof Of Work based coins. Currently, there is no production build of the c2pool.

**Comment by pfreak2000** [comment]
uff... OK
so i must switch back to p2pool and find anyone for ravencoin iomplementation

**Comment by frstrtr** [comment]
https://github.com/jtoomim/p2pool is the most recent p2pool at this time, you can ask there...

**Comment by pfreak2000** [comment]
thanks for help :-)

**Comment by frstrtr** [comment]
I think RavenCoin p2pool implementation is possible to be developed by our team, but for a price...
